go/libraries/doltcore/sqle/dprocedures/dolt_verify_constraints.go

```diff
@@ -75,37 +77,110 @@ func doDoltConstraintsVerify(ctx *sql.Context, args []string) (int, error) {
 		}
 	}
 
-	tableSet := set.NewStrSet(nil)
-	for _, val := range apr.Args {
-		_, tableName, ok, err := workingRoot.GetTableInsensitive(ctx, val)
-		if err != nil {
-			return 1, err
-		}
-		if !ok {
-			return 1, sql.ErrTableNotFound.New(tableName)
-		}
-		tableSet.Add(tableName)
+	tableSet, err := parseTablesToCheck(ctx, workingRoot, apr)
+	if err != nil {
+		return 1, err
 	}
 
-	newRoot, tablesWithViolations, err := merge.AddForeignKeyViolations(ctx, workingRoot, comparingRoot, tableSet, h)
+	// Check for all non-FK constraint violations
+	newRoot, tablesWithViolations, err := calculateViolations(ctx, workingRoot, comparingRoot, tableSet)
 	if err != nil {
 		return 1, err
 	}
 
+	if !outputOnly {
+		err = dSess.SetRoot(ctx, dbName, newRoot)
+		if err != nil {
+			return 1, err
+		}
+	}
+
 	if tablesWithViolations.Size() == 0 {
 		// no violations were found
 		return 0, nil
 	}
 
-	// violations were found
+	// TODO: We only return 1 or 0 to indicate if there were any constraint violations or not. This isn't
```

Comment:
Seems like a fast-follow

Reply:
Yeah, this is an easy one to change, but before changing the behavior, I think we should look at migrating `dolt constraints verify` over to use the stored procedure – I suspect there will be other change to the result data that we'll need to make to support that.